resource/user/user.go

```diff
@@ -401,6 +336,40 @@ func (u *User) DiffAdd(status *resource.Status) (*AddUserOptions, error) {
 	return options, nil
 }
 
+// DiffDel checks for differences between the current and desired state for the
+// user to be deleted indicated by the User fields.
+func (u *User) DiffDel(status *resource.Status, userByName *user.User, nameNotFound bool) error {
```

Comment:
After some discussion, we've concluded to leave as is for `user` since testing is currently performed for a `package <name>` in `package <name_test>`. In the next resource added to  Converge, we will try testing as part of the package.